technical/standalone-mvt.md

```diff
@@ -1,3 +1,5 @@
+# **TO BE DEPRECATED, PLEASE SPEAK TO THE TASTER TEAM FOR INFORMATION (tastertech@rd.bbc.co.uk)**
```

Comment:
I think we should say that this _is_ deprecated rather than it is _to be_ deprecated.